### DISCUSSION TITLE
Progress?

### DISCUSSION THREAD

**Discussion by RetroGameTalk** [furiousg4m3r/Tales-of-Destiny-DC#99] [General]
Hi, thanks for this.
I'd like to know what is translated on 0.0.8?
Menus? Items? Story? A % of some?
Thanks!

**Comment by pnvnd** [comment]
@MediaMaquina Someone on the Discord server replied:
- story = 1%
- skit = 0.001%
- items = 60%
- menu = 40%

**Reply by RetroGameTalk** [reply]
Sorry can this be updated on the readme after some number of releases? To get a feel of what's new?

**Comment by RetroGameTalk** [comment]
Thanks!

**Comment by H0R1Z0N** [comment]
I had subs to email notification for when there was a commit or merge, i see you guys keep pushing commit and merge in the last weekend, on one side, the notification start to be annoying for the first time, but i am like "OH DANG, THAT PATCH MIGHT GET RELEASE BEFORE THE END OF THE YEARS ! :grinning: I wish you a good holiday, that make me happy for seeing how much you guys are passionate about this project to make it possible. Keep on the good work, i cannot wait to see the last notification about a new release (Which you said was gonna be final 1.0), which would be super awesome. Keep on rocking.

**Comment by pnvnd** [comment]
```
 _     _  __        ____        _   _   _      
| |   (_)/ _| ___  | __ )  ___ | |_| |_| | ___ 
| |   | | |_ / _ \ |  _ \ / _ \| __| __| |/ _ \
| |___| |  _|  __/ | |_) | (_) | |_| |_| |  __/
|_____|_|_|  \___| |____/ \___/ \__|\__|_|\___|
```
